src/test/java/service/strategy/ReturnOperationHandlerTest.java

```diff
@@ -0,0 +1,37 @@
+package service.strategy;
+
+import static org.junit.Assert.assertEquals;
+
+import dao.StorageDaoImpl;
+import db.Storage;
+import model.Fruit;
+import model.FruitTransaction;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ReturnOperationHandlerTest {
+    private static OperationHandler operationHandler;
+
+    @Before
+    public void setUp() {
+        operationHandler = new ReturnOperationHandler(new StorageDaoImpl());
+    }
+
+    @Test
+    public void apply_correctTransaction_Ok() {
+        Fruit apple = new Fruit("apple");
+        Storage.dataBase.put(apple, 40);
+        FruitTransaction fruitTransaction =
+                new FruitTransaction(FruitTransaction.Operation.PURCHASE, apple, 32);
```

Comment:
```suggestion
                new FruitTransaction(FruitTransaction.Operation.RETURN, apple, 32);
```
ти в усіх хенделерах передаєш операцію покупки, виправ це в усіх тестових класах